model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,55 @@
+groups:
+  - id: mobile-crash-event
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'device' and `event.name` is 'crash
+    attributes:
+      - ref: os.name
```

Comment:
I think we can omit `os.name` here because `os.name` will be capture in the resource attributes anyways.